<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMessage</class>
 <widget class="QMainWindow" name="UIMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messagerie Interne - Administrateur</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidgetMessages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabNewMessage">
       <attribute name="title">
        <string>Nouveau Message</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelReceiver">
            <property name="text">
             <string>Destinataire:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxReceiver">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSubject">
            <property name="text">
             <string>Objet:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditSubject"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelMessage">
          <property name="text">
           <string>Message:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditMessageContent"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSend">
            <property name="text">
             <string>Envoyer</string>
            </property>
            <property name="icon">
             <iconset theme="mail-send">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonQuit">
            <property name="text">
             <string>Quitter</string>
            </property>
            <property name="icon">
             <iconset theme="cancel">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabReceived">
       <attribute name="title">
        <string>Messages Reçus</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableViewReceived">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSent">
       <attribute name="title">
        <string>Messages Envoyés</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tableViewSent">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonRefreshSent">
            <property name="text">
             <string>Actualiser</string>
            </property>
            <property name="icon">
             <iconset theme="view-refresh">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDeleteSent">
            <property name="text">
             <string>Supprimer</string>
            </property>
            <property name="icon">
             <iconset theme="edit-delete">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDetail">
       <attribute name="title">
        <string>Détail du message</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSender">
            <property name="text">
             <string>De:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelSenderValue">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelDate">
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelDateValue">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSubjectDetail">
            <property name="text">
             <string>Objet:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelSubjectValue">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelContent">
          <property name="text">
           <string>Contenu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserContent"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau_message"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuMessages">
    <property name="title">
     <string>Messages</string>
    </property>
    <addaction name="actionActualiser"/>
    <addaction name="actionSupprimer"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuMessages"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNouveau_message">
   <property name="text">
    <string>Nouveau message</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionActualiser">
   <property name="text">
    <string>Actualiser</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
